core/src/main/java/org/stellar/anchor/sep12/Sep12Service.java

```diff
@@ -68,20 +68,24 @@ public void deleteCustomer(JwtToken jwtToken, String account, String memo, Strin
           String.format("Not authorized to delete account [%s]", account));
     }
 
-    Sep12GetCustomerResponse existingCustomer =
-        customerIntegration.getCustomer(
-            Sep12GetCustomerRequest.builder()
-                .account(account)
-                .memo(memo)
-                .memoType(memoType)
-                .build());
-    if (existingCustomer.getId() == null) {
-      infoF(
-          "No existing customer found for account={} memo={} memoType={}", account, memo, memoType);
-      throw new SepNotFoundException("User not found.");
-    }
+    String senderTypes[] = {"sep31-sender", "sep31-receiver"};
```

Comment:
```suggestion
    String customerTypes[] = {"sep31-sender", "sep31-receiver"};
```

Reply:
Maybe for another PR, move it to a config file.

Reply:
Acknowledged, added a TODO for this